File: Javier_Programacion/TrabajoFinalJSP/src/main/java/com/daw2final/trabajofinaljsp/model/entity/Proveedor.java
package com.daw2final.trabajofinaljsp.model.entity;

import java.util.List;

public class Proveedor {
    private Integer id;
    private String nif;
    private String nombre;
    private String apellido1;
    private String apellido2;
    private String telefono;
    private String razonSocial;
    private String email;
    protected List<Articulo> articulos;

    public Proveedor() {
        this(null,null,null,null,null,null,null,null);
    }
    public Proveedor(Integer id, String nif, String nombre, String apellido1, double telefono, String razonSocial, String email) {
        this(null,null,null,null,null,null,null,null);
    }

    public Proveedor(String nif, String nombre, String apellido1, String apellido2, String telefono, String razonSocial, String email) {
        this.id = null;
        this.nif = nif;
        this.nombre = nombre;
        this.apellido1 = apellido1;
        this.apellido2 = apellido2;
        this.telefono = telefono;
        this.razonSocial = razonSocial;
        this.email = email;
    }
    public Proveedor(Integer id, String nif, String nombre, String apellido1, String apellido2, String telefono, String razonSocial, String email) {
        this.id = id;
        this.nif = nif;
        this.nombre = nombre;
        this.apellido1 = apellido1;
        this.apellido2 = apellido2;
        this.telefono = telefono;
        this.razonSocial = razonSocial;
        this.email = email;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNif() {
        return nif;
    }

    public void setNif(String nif) {
        this.nif = nif;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido1() {
        return apellido1;
    }

    public void setApellido1(String apellido1) {
        this.apellido1 = apellido1;
    }

    public String getApellido2() {
        return apellido2;
    }

    public void setApellido2(String apellido2) {
        this.apellido2 = apellido2;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getRazonSocial() {
        return razonSocial;
    }

    public void setRazonSocial(String razonSocial) {
        this.razonSocial = razonSocial;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public List<Articulo> getArticulos() {
        return articulos;
    }

    public void setArticulos(List<Articulo> articulos) {
        this.articulos = articulos;
    }

    @Override
    public String toString() {
        return "Proveedor{" +
                "id=" + id +
                ", nif='" + nif + '\'' +
                ", nombre='" + nombre + '\'' +
                ", apellido1='" + apellido1 + '\'' +
                ", apellido2='" + apellido2 + '\'' +
                ", telefono=" + telefono +
                ", razonSocial='" + razonSocial + '\'' +
                ", email='" + email + '\'' +
                '}';
    }
}

File: ERPDAW2-Juanjo/pojos/Proveedor.php
<?php

class Proveedor{
    private $id;
    private $idProveedor;
    private $nombre;
    private $apellido1;
    private $apellido2;
    private $nif;
    private $codPostal;
    private $localidad;
    private $provincia;
    private $activo;
    

    public function __construct($id, $idProveedor, $nombre, $apellido1, $apellido2, $nif, $codPostal, $localidad, $provincia, $activo)
    {
        $this->id = $id;
        $this->idProveedor = $idProveedor;
        $this->nombre = $nombre;
        $this->apellido1 = $apellido1;
		$this->apellido2 = $apellido2;
		$this->nif = $nif;
        $this->codPostal = $codPostal;
        $this->localidad = $localidad;
        $this->provincia = $provincia;
        $this->activo = $activo;
    }

    
    public function getId()
    {
        return $this->id;
    }

    
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }
    
    public function getIdProveedor()
    {
        return $this->idProveedor;
    }

    
    public function setIdProveedor($idProveedor)
    {
        $this->idProveedor = $idProveedor;

        return $this;
    }

    
    public function getNombre()
    {
        return $this->nombre;
    }

    
    public function setNombre($nombre)
    {
        $this->nombre = $nombre;

        return $this;
    }

    
    public function getApellido1()
    {
        return $this->apellido1;
    }

    
    public function setApellido1($apellido1)
    {
        $this->apellido1 = $apellido1;

        return $this;
    }

    
    public function getApellido2()
    {
        return $this->apellido2;
    }

    
    public function setApellido2($apellido2)
    {
        $this->apellido2 = $apellido2;

        return $this;
    }

    
    public function getNif()
    {
        return $this->nif;
    }

    
    public function setNif($nif)
    {
        $this->nif = $nif;

        return $this;
    }

    
    public function getCodPostal()
    {
        return $this->codPostal;
    }

    
    public function setCodPostal($codPostal)
    {
        $this->codPostal = $codPostal;

        return $this;
    }

   
    public function getLocalidad()
    {
        return $this->localidad;
    }

    
    public function setLocalidad($localidad)
    {
        $this->localidad = $localidad;

        return $this;
    }

    
    public function getProvincia()
    {
        return $this->provincia;
    }

    
    public function setProvincia($provincia)
    {
        $this->provincia = $provincia;

        return $this;
    }

    
    public function getActivo()
    {
        return $this->activo;
    }

    
    public function setActivo($activo)
    {
        $this->activo = $activo;

        return $this;
    }
}
?>